consensus/state.go

```diff
@@ -877,7 +877,7 @@ func (cs *State) handleMsg(mi msgInfo) {
 		// We probably don't want to stop the peer here. The vote does not
 		// necessarily comes from a malicious peer but can be just broadcasted by
 		// a typical peer.
-		// https://github.com/tendermint/tendermint/issues/1281
+		// https://github.com/cometbft/cometbft/issues/1281
```

Comment:
fwiw I'm don't think its worth going through and finding each broken link as I don't think they're super useful and we can just manually fix the links if we really need to